### PR TITLE
FS-3718 Update tag version generation to be unified

### PR DESCRIPTION
**Pull request by aaronwilliamsv1**
### Change description

* Making the tagging version match with the image version referenced

+ref: FS-3718
+semver: minor

- [N/A] Unit tests and other appropriate tests added or updated
- [N/A] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### CODE REVIEW

**Review by gidsg — DISMISSED** [review]
LGTM 👍

**Review by robk-dluhc — APPROVED** [review]
🧀

File: .github/workflows/copilot_deploy.yml [hunk 1]
```diff
@@ -106,15 +118,15 @@ jobs:
...
+        yq -i '.image.location = "ghcr.io/communitiesuk/funding-service-design-notification:${{ github.ref_name == 'main' && 'latest' || needs.tag_version.outputs.version_to_tag }}"'  copilot/fsd-notification/manifest.yml
 
     - name: Copilot ${{ matrix.value }} deploy
       id: deploy_build
       run: |
         copilot svc deploy --env ${{ matrix.value }} --app pre-award
 
   post_deploy_tests:
-    if: inputs.environment == 'test' || inputs.environment == ''
+    if: inputs.environment == 'test' || inputs.environment == 'dev' || inputs.environment == ''
```
Comment (robk-dluhc) [inline .github/workflows/copilot_deploy.yml:129]: If we deploy to production is this going to run post-deploy tests there?  Is that what we want?

Comment (aaronwilliamsv1) [reply .github/workflows/copilot_deploy.yml:129]: Hmm, cos of line 139, I guess it would do. No, we don't.

**Review by robk-dluhc — DISMISSED** [review]

**Review by robk-dluhc — APPROVED** [review]
🇦🇱

**Pull request merged**